modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -25,6 +25,11 @@ ifdef::openshift-dedicated,openshift-rosa[]
 endif::openshift-dedicated,openshift-rosa[]
 * You have installed the OpenShift CLI (`oc`).
 * You have set up a remote write compatible endpoint (such as Thanos) and know the endpoint URL. See the link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Prometheus remote endpoints and storage documentation] for information about endpoints that are compatible with the remote write feature.
++
+[IMPORTANT]
+====
+Red Hat only provides information for configuring remote write senders and does not offer guidance on configuring receiver endpoints. Endpoint receivers are not included in Red Hat production support and therefore a user is responsible to set up remote write compatible endpoints.
```

Comment:
```suggestion
Red Hat only provides information for configuring remote write senders and does not offer guidance on configuring receiver endpoints. Customers are responsible for setting up their own remote write compatible endpoints. Issues with endpoint receivers are not included in Red Hat production support.
```
- First sentence looks great! Thanks.

A couple of other suggestions:

- Add a comma before the coordinating conjunction `support, and` if you keep the current syntax and sentence structure, but I think it provides better logical flow to break that long second sentence into two shorter sentences. Ending the sentence by mentioning lack of RH production support really helps drive home the point, too. WDYT?
- I think it's better to say customer here rather than user, because we are really talking more generically about customer support issues rather than actions by individual users. WDYT?
- Revise for idiomatic use of preposition following `is responsible`: `is responsible for setting up`.